docs/admin.md

```diff
@@ -0,0 +1,88 @@
+(TODO: link to the diagram here) 
+
+Mapping of configMap keys (yaml files) to runtime objects (NOTE: for the time (Dec 20'23) it is a subject of change):
+| Key/File name         | k8s/OCP Kind        | Mandatory*    |Notes                                      |
+| ----------------------|:-------------------:| --------------|------------------------------------------:|
+| deployment.yaml       | appsv1.Deployment   | Yes           | Backstage deployment |
+| service.yaml          | corev1.Service      | Yes           | Backstage Service |
+| db-statefulset.yaml   | appsv1.Statefulset  | For DB enabled| PostgreSQL statefulSet    |    
+| db-service.yaml       | corev1.Service      | For DB enabled| PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled| Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  | Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            | Backstage app-config.yaml    |
+| configmap-files.yaml  | corev1.ConfigMap    | No            | Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            | Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            | Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            | Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            | dynamc-plugins config *    |
+
+
+NOTES: 
+ - Mandatory means it is needed to be present in either (or both) Default and CR Raw Configuration.
+ - dynamic-plugins.yaml is a fragment of app-config.yaml provided with RHDH/Janus-IDP. The reason it mentioned separately is specific way it provided (via mounting to dedicated initContainer)  
+
+### Operator Bundle configuration 
+
+With Backstage Operator's Makefile you can generate bundle descriptor using *make bundle* command
+
+Along with CSV manifest it generates default-config ConfigMap manifest, which can be modified and applied to Backstage Operator
+TODO: is kubectl sufficient or there are some other tools?
+
+### Kustomize deploy configuration
+
+Make sure use the current context in your kubeconfig file is pointed to correct place, change necessary part of your config/manager/default-config or just replace some of the file(s) with yours and run
+``
+make deploy
+``
+
+### Direct ConfigMap configuration
+
+You can change default configuration by directly changing the default-config ConfigMap with kubectl like:
+
+ - if you need to get existed default-config
+
+``
+kubectl get -n backstage-system configmap default-config > my-config.yaml
+``
+
+- modify any part of it (yes, it could be fairly big file)
+
+- apply it back
+
+``
+  kubectl apply -n backstage-system -f my-config.yaml
+``
+
+It has to be re-applied to the controller's container after some time.
```

Comment:
"after some time" ... how long? why? do you mean it'll be overwritten when the operator subscription is updated from 1.1 -> 1.2?

Reply:
when k8s reconciles it.